apps/cowswap-frontend/src/modules/swap/helpers/getAmountsForSignature.ts

```diff
@@ -0,0 +1,73 @@
+import { ONE_FRACTION } from '@cowprotocol/common-const'
+import { OrderKind } from '@cowprotocol/cow-sdk'
+import { Currency, CurrencyAmount, Percent, TradeType } from '@uniswap/sdk-core'
+
+import TradeGp from 'legacy/state/swap/TradeGp'
+
+interface AmountForSignatureParams {
+  trade: TradeGp
+  allowedSlippage: Percent
+  kind: OrderKind
+  featureFlags: {
```

Comment:
nit: do we expect a lot of feature flags for this utility function? a simpler flag here would have been enough. But is also fine! happy to leave